Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -305,8 +305,14 @@ public class ApolloCodegen {
     )
 
     guard graphqlErrors.isEmpty else {
-      let errorlines = graphqlErrors.flatMap({ $0.logLines })
-      CodegenLogger.log(String(describing: errorlines), logLevel: .error)
+      let errorlines = graphqlErrors.flatMap({
+        if let logLines = $0.logLines {
+          return logLines
+        } else {
+          return ["[Error] \($0.name ?? "unknown"): \($0.message ?? "")"]
```

Comment:
I believe the `[Error]` label is already prepended to the log by the `CodegenLogger`. So putting it here might make it display it twice in the actual log.

Reply:
Good callout I'll remove that